scripts/package.ps1

```diff
@@ -0,0 +1,54 @@
+#Requires -Version 5.0
+$ErrorActionPreference = "Stop"
+
+Invoke-Expression -Command "$PSScriptRoot\version.ps1"
+
+$DIR_PATH = Split-Path -Parent $MyInvocation.MyCommand.Definition
+$SRC_PATH = (Resolve-Path "$DIR_PATH\..").Path
+cd $SRC_PATH\package\windows
+
+
+$TAG = $env:TAG
+if (-not $TAG) {
+    $TAG = ('{0}{1}' -f $env:VERSION, $env:SUFFIX)
+}
+$REPO = $env:REPO
+if (-not $REPO) {
+    $REPO = "rancher"
+}
+
+if ($TAG | Select-String -Pattern 'dirty') {
+    $TAG = "dev"
+}
+
+if ($env:DRONE_TAG) {
+    $TAG = $env:DRONE_TAG
+}
+
+# Get release id as image tag suffix
+$HOST_RELEASE_ID = (Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\' -ErrorAction Ignore).ReleaseId
```

Comment:
this release id stuff doesnt work any more, it was an idea a while back for running docker in hyperv and doesnt work with how we have have drone hosts today. you can probably ditch all this logic and just have the else statement below. we've done this before in fleet

Reply:
dropped in latest commit. Let me know if the change looks good to you and I can squash